packages/admin/admin-generator/src/commands/generate/generate-command.ts

```diff
@@ -184,7 +185,26 @@ export type GridConfig<T extends { __typename?: string }> = {
     selectionProps?: "multiSelect" | "singleSelect";
 };
 
-export type GeneratorConfig<T extends { __typename?: string }> = FormConfig<T> | GridConfig<T> | TabsConfig<T>;
+type CrudPageGridConfig = {
+    component: ComponentType;
+};
+
+export type CrudPageFormConfig = {
+    component: ComponentType;
+    pageTitle?: string;
+};
+
+export type CrudPageConfig<T extends { __typename?: string }> = {
+    type: "crudPage";
+    topLevelTitle?: string;
+    gqlType: T["__typename"];
+    grid: CrudPageGridConfig;
+    forms?: CrudPageFormConfig;
+    addForm?: Partial<CrudPageFormConfig>;
+    editForm?: Partial<CrudPageFormConfig>;
```

Comment:
why partial?

Reply:
My though was you could use `forms` to configure both forms at once for using the same form. 
Then, using `addForm`/`editForm` you could optionally override individual settings, e.g. if you want one form to have a specific `pageTitle`.

Reply:
I understand, but I think we need to improve that api

Reply:
Improve it in what way exactly?

Reply:
Maybe `form?: Partial<CrudPageFormConfig> | { addForm?: Partial<CrudPageFormConfig>; editForm?: Partial<CrudPageFormConfig> }`?